### DISCUSSION TITLE
Missed reading alert not working

### DISCUSSION THREAD

**Discussion by Jontellekemp** [NightscoutFoundation/xDrip#1944] [General]
Hi guys,

Im trying to get my sons missed readings alert to alarm on his follower phone and haven’t found a resolution yet. 

Any help, much appreciated. 

https://www.facebook.com/groups/xDripG5/permalink/6758375057569265/

**Comment by Navid200** [comment]
There are developers who don't have facebook accounts.  The reason I suggested you post here was to hopefully get hints from them.
For this to work, you need to enter the important posts here instead of placing a link to facebook.

You can add posts to this thread and copy your important posts from facebook here.

I asked you lots of questions there.  You can post all your answers here.
